src/app/app.component.html

```diff
@@ -0,0 +1,6 @@
+<app-header [title]="title"></app-header>
```

Comment:
You don't need to move the title as an input. Write the title in the header component